src/program.js

```diff
@@ -447,15 +447,15 @@ Example: $0 --help run.
                   'preference.',
         demand: false,
         requiresArg: true,
-        type: 'string',
+        type: 'array',
         coerce: coerceCLICustomPreference,
       },
       'start-url': {
         alias: ['u', 'url'],
         describe: 'Launch firefox at specified page',
         demand: false,
         requiresArg: true,
-        type: 'string',
+        type: 'array',
```

Comment:
This Flow type needs an update too 

https://github.com/rpl/web-ext/blob/3848c777b290b4c5ed6ad1af0436afe5f952b005/src/cmd/run.js#L43